src/jfr/jenkinsfile-runner.inc

```diff
@@ -8,6 +8,9 @@ set -e
 run_jfr_docker_image() {
 
     if [ "$#" -gt 1 ]
+    then
+        docker run -e JAVA_OPTS="$JAVA_OPTS $3" --rm -v "$2":/workspace/Jenkinsfile "$1"
```

Comment:
This will be executed even if `JAVA_OPTS` is not set and `$3` is not passed since with only `$1` and `$2` the condition will return true. Will it work in that case? If so, maybe the function only should execute this command and we can delete my if below: https://github.com/jenkinsci/jenkinsfile-runner-test-framework/pull/6/files#diff-93253b70ce0abb644c69126e61809ff5R15